scrape/scrape_test.go

```diff
@@ -744,15 +716,10 @@ func TestScrapeLoopRunCreatesStaleMarkersOnFailedScrape(t *testing.T) {
 
 	// 1 successfully scraped sample, 1 stale marker after first fail, 5 report samples for
 	// each scrape successful or not.
-	if len(appender.result) != 27 {
-		t.Fatalf("Appended samples not as expected. Wanted: %d samples Got: %d", 27, len(appender.result))
-	}
-	if appender.result[0].v != 42.0 {
-		t.Fatalf("Appended first sample not as expected. Wanted: %f Got: %f", appender.result[0].v, 42.0)
-	}
-	if !value.IsStaleNaN(appender.result[6].v) {
-		t.Fatalf("Appended second sample not as expected. Wanted: stale NaN Got: %x", math.Float64bits(appender.result[5].v))
-	}
+	testutil.Equals(t, 27, len(appender.result), "Appended samples not as expected")
+	testutil.Equals(t, 42.0, appender.result[0].v, "Appended first sample not as expected")
+	testutil.Assert(t, value.IsStaleNaN(appender.result[6].v),
+		"Appended second sample not as expected. Wanted: stale NaN Got: %x", math.Float64bits(appender.result[5].v))
```

Comment:
Possibly a bug in the previous code, but shouldn't both be `result[6]`?

Reply:
Yes, sure. Fixed that.